continuous_integration/recipe/meta.yaml

```diff
@@ -37,6 +39,8 @@ requirements:
     - pygments
     - nest-asyncio
     - tabulate
+    - setuptools-rust>=1.1.2
```

Comment:
Out of interest, what happens if we don't have `setuptools-rust` installed along with the conda package? Interested in why we need it as a runtime and host dependency

Reply:
Since the `setup.py` file has references to `Rust` extensions the build will fail if setuptools-rust is not present.

Reply:
Oh, I was mistaken and thought this was another file. Yes, this can be removed from this section and is in my latest commit. Thanks for catching that and I will resolve this.